### PR TITLE
[cluster-test] fix account num limitation for CT running on premainnet

### PR DESCRIPTION
**Pull request by snakech**
We do check account number for each parent VASP to avoid exceed limit, but for premainnet we do not need this since we only support 1 parent vasp account.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan
Reproduce issue before fix, on rev 7714794f11:
~/libra-core/target/debug/cluster-test --swarm --emit-tx --premainnet --chain-id=18 --peers=a72ffeb94e81111ea81b70e75131a869-1894457723.ap-northeast-1.elb.amazonaws.com:80,34.68.123.37:80,ada61ba47e80d11ea9bcd0a7a792c33e-1496143175.ap-southeast-1.elb.amazonaws.com:80,34.73.243.120:80,libra-breakthrough-full-node.man.bdi.sh:80,ab6b29969132f484db9fdc9d302b92e1-9b15e1987ad0319f.elb.eu-west-1.amazonaws.com:80,34.74.8.217:80,20.50.12.245:80,195.154.71.193:80,ad49a6c1e472c4f82ac8507df2d81f58-e2858d056c4dfef9.elb.us-east-2.amazonaws.com:80,aeb83d6e0bf8f46f7bb25dd68c82f021-db6448367d0c7b7b.elb.us-east-1.amazonaws.com:80,35.198.155.232:80,premainnet.libra.novi.com:80,ae5dfb87ce81311ea8c230282fe68c3b-1804125671.us-west-2.elb.amazonaws.com:80,34.107.49.2:80,35.241.88.188:80,34.70.43.25:80,libra-slow-full-node.man.bdi.sh:80,104.197.155.200:80,libra-temasek-full-node.man.bdi.sh:80,a9a46a9e8c0aa4ab0995a36de953d2f5-68825d4ac845d301.elb.us-west-2.amazonaws.com:80,34.73.15.170:80,a4d1bf61ae81311eaaaf002ebf759cd5-341763703.us-east-2.elb.amazonaws.com:80,a8c8481bcc817472581e91a468c671da-44b7ecf30878f348.elb.eu-west-1.amazonaws.com:80 --duration=300 --accounts-per-client=1 --workers-per-ac=10

```
2020-10-12T17:28:03.532747Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:234 Will use 10 workers per AC with total 240 AC clients
2020-10-12T17:28:03.532811Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:246 Will create 1 accounts_per_client with total 240 accounts
2020-10-12T17:28:03.533054Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:387 Loading faucet account from DD account
2020-10-12T17:28:03.698233Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:393 DD account current balances are 499997115000000, requested 240000000 coins
2020-10-12T17:28:03.698519Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:425 Loading VASP account as seed accounts
2020-10-12T17:28:06.142611Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:468 Completed minting seed accounts
2020-10-12T17:28:06.142677Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:469 Minting additional 240 accounts
2020-10-12T17:28:06.143169Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1064 loading 10 accounts if they exist
thread 'main' panicked at 'Something wrong in mint_account, wanted to mint 240, only have 10', testsuite/cluster-test/src/tx_emitter.rs:500:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

With fix:

- Test with local swarm:
cargo run -p cluster-test -- --mint-file "/private/var/folders/qd/tg50mr0d7k75qvztr_8yk2km0000gn/T/823ff4d3a27d5b38581d782dcf675b43/mint.key" --swarm --peers "127.0.0.1:8080" --emit-tx --chain-id TESTING
```
2020-10-12T19:04:25.205819Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:234 Will use 10 workers per AC with total 10 AC clients
2020-10-12T19:04:25.205885Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:246 Will create 15 accounts_per_client with total 150 accounts
2020-10-12T19:04:25.206216Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:375 Creating and minting faucet account
2020-10-12T19:04:25.870779Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:393 DD account current balances are 9223372036854775807, requested 150000000 coins
2020-10-12T19:04:25.870941Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:410 Creating and minting seeds accounts
2020-10-12T19:04:26.529216Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:421 Completed creating seed accounts
2020-10-12T19:04:27.179457Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:469 Completed minting seed accounts
2020-10-12T19:04:27.179486Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:470 Minting additional 150 accounts
2020-10-12T19:04:36.864693Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:507 Mint is done
2020-10-12T19:04:36.865725Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:280 Tx emitter workers started
2020-10-12T19:04:46.868241Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 63 txn/s, committed: 48 txn/s, expired: 0 txn/s, latency: 2402 ms, p99 latency: 2550 ms
2020-10-12T19:04:56.871787Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 72 txn/s, committed: 72 txn/s, expired: 0 txn/s, latency: 2248 ms, p99 latency: 2400 ms
2020-10-12T19:05:06.872586Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 60 txn/s, committed: 60 txn/s, expired: 0 txn/s, latency: 2301 ms, p99 latency: 2450 ms
2020-10-12T19:05:16.876331Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 60 txn/s, committed: 60 txn/s, expired: 0 txn/s, latency: 2278 ms, p99 latency: 2400 ms
2020-10-12T19:05:26.879473Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 60 txn/s, committed: 60 txn/s, expired: 0 txn/s, latency: 2320 ms, p99 latency: 2450 ms
2020-10-12T19:05:36.880702Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 60 txn/s, committed: 60 txn/s, expired: 0 txn/s, latency: 2310 ms, p99 latency: 2550 ms
Total stats: submitted: 3750, committed: 3750, expired: 0
Average rate: submitted: 62 txn/s, committed: 62 txn/s, expired: 0 txn/s, latency: 2318 ms, p99 latency: 2600 ms
```

- Test with premainnet:
./target/debug/cluster-test --swarm --emit-tx --premainnet --chain-id=18 --peers=a72ffeb94e81111ea81b70e75131a869-1894457723.ap-northeast-1.elb.amazonaws.com:80,34.68.123.37:80,ada61ba47e80d11ea9bcd0a7a792c33e-1496143175.ap-southeast-1.elb.amazonaws.com:80,34.73.243.120:80,libra-breakthrough-full-node.man.bdi.sh:80,ab6b29969132f484db9fdc9d302b92e1-9b15e1987ad0319f.elb.eu-west-1.amazonaws.com:80,34.74.8.217:80,20.50.12.245:80,195.154.71.193:80,ad49a6c1e472c4f82ac8507df2d81f58-e2858d056c4dfef9.elb.us-east-2.amazonaws.com:80,aeb83d6e0bf8f46f7bb25dd68c82f021-db6448367d0c7b7b.elb.us-east-1.amazonaws.com:80,35.198.155.232:80,premainnet.libra.novi.com:80,ae5dfb87ce81311ea8c230282fe68c3b-1804125671.us-west-2.elb.amazonaws.com:80,34.107.49.2:80,35.241.88.188:80,34.70.43.25:80,libra-slow-full-node.man.bdi.sh:80,104.197.155.200:80,libra-temasek-full-node.man.bdi.sh:80,a9a46a9e8c0aa4ab0995a36de953d2f5-68825d4ac845d301.elb.us-west-2.amazonaws.com:80,34.73.15.170:80,a4d1bf61ae81311eaaaf002ebf759cd5-341763703.us-east-2.elb.amazonaws.com:80,a8c8481bcc817472581e91a468c671da-44b7ecf30878f348.elb.eu-west-1.amazonaws.com:80 --accounts-per-client=1 --workers-per-ac=10

```
2020-10-12T19:00:32.889019Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:234 Will use 10 workers per AC with total 240 AC clients
2020-10-12T19:00:32.889086Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:246 Will create 1 accounts_per_client with total 240 accounts
2020-10-12T19:00:32.889396Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:387 Loading faucet account from DD account
2020-10-12T19:00:33.516926Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:393 DD account current balances are 499995935000000, requested 240000000 coins
2020-10-12T19:00:33.517190Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:425 Loading VASP account as seed accounts
2020-10-12T19:00:36.331505Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:469 Completed minting seed accounts
2020-10-12T19:00:36.331534Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:470 Minting additional 240 accounts
2020-10-12T19:00:36.331848Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:00:44.050602Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:00:51.069706Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:00:59.143497Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:09.497549Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:19.951636Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:28.337513Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:38.075584Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:46.415563Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:01:54.796754Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:02:02.765952Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:02:09.608387Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:1065 loading 240 accounts if they exist
2020-10-12T19:02:16.215360Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:507 Mint is done
2020-10-12T19:02:16.256056Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:280 Tx emitter workers started
2020-10-12T19:02:26.257066Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 128 txn/s, committed: 104 txn/s, expired: 0 txn/s, latency: 2060 ms, p99 latency: 3650 ms
2020-10-12T19:02:36.257962Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 115 txn/s, committed: 115 txn/s, expired: 0 txn/s, latency: 2067 ms, p99 latency: 4450 ms
2020-10-12T19:02:46.259059Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 123 txn/s, committed: 123 txn/s, expired: 0 txn/s, latency: 1985 ms, p99 latency: 4950 ms
2020-10-12T19:02:56.260120Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 116 txn/s, committed: 116 txn/s, expired: 0 txn/s, latency: 2032 ms, p99 latency: 6500 ms
2020-10-12T19:03:06.261073Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 118 txn/s, committed: 118 txn/s, expired: 0 txn/s, latency: 2039 ms, p99 latency: 4700 ms
2020-10-12T19:03:16.262172Z [main] INFO testsuite/cluster-test/src/tx_emitter.rs:536 submitted: 114 txn/s, committed: 114 txn/s, expired: 0 txn/s, latency: 2051 ms, p99 latency: 4200 ms
Total stats: submitted: 7160, committed: 7160, expired: 0
Average rate: submitted: 119 txn/s, committed: 119 txn/s, expired: 0 txn/s, latency: 2040 ms, p99 latency: 4450 ms

```

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: testsuite/cluster-test/src/tx_emitter.rs [hunk 1]
```diff
@@ -439,22 +439,23 @@ impl TxEmitter {
...
         let coins_per_account = (SEND_AMOUNT + req.gas_price) * MAX_TXNS;
-        let coins_per_seed_account = coins_per_account * num_new_child_accounts as u64;
         let coins_total = coins_per_account * num_accounts as u64;
 
         let mut faucet_account = self.get_money_source(&req.instances, coins_total).await?;
         // Create seed accounts with which we can create actual accounts concurrently
         let seed_accounts = self
             .get_seed_accounts(&req.instances, num_seed_accounts)
             .await?;
+        num_seed_accounts = seed_accounts.len();
```
Comment (davidiw) [inline testsuite/cluster-test/src/tx_emitter.rs:456]: without any comments this is hard to read.... it looks like the first value is `expected_num_seed_accounts` and the second value is `actual_num_seed_accounts` could you us either two variables or leavea comment on why we're resetting this value?

**Review by davidiw — APPROVED** [review]

File: testsuite/cluster-test/src/tx_emitter.rs (hunk 1 above)
Comment (snakech) [reply testsuite/cluster-test/src/tx_emitter.rs:456]: oh, yea, I am going to add some comments

Comment (snakech) [reply testsuite/cluster-test/src/tx_emitter.rs:456]: because for premainnet, the seed account num is always 1.

**Pull request closed without merging**